{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "python3",
                "purl": "pkg:rpm/vmware/python3?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "3.6.9-6.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "elasticsearch",
                "purl": "pkg:rpm/vmware/elasticsearch?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "6.8.13-1.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2021-0317",
    "modified": "2023-03-16T05:25:30Z",
    "published": "2021-02-11T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-2.0-317"
        }
    ],
    "related": [
        "CVE-2021-3177",
        "CVE-2020-7020"
    ]
}